editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -284,9 +291,12 @@ private void addDescriptionAsTooltip(Parameter parameter, Label label) {
 	 * @return
 	 */
 	protected CamelModel getCamelModel(AbstractCamelModelElement modelElement) {
-		CamelModel model = modelElement.getCamelFile().getCamelModel();
-		if (model == null) {
-			return null;
+		CamelModel model = null;
+		if (modelElement != null) {
+			model = modelElement.getCamelFile().getCamelModel();
+		}
+		if (model == null && lastSelectedEP != null) {
```

Comment:
I really feel that it is a hack.
Why we are not able to provide a non-null modelElement?